.docker/stable.Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN DESTDIR=/install cmake --build /build -- install
 
 FROM greenbone/community-feed-vts AS feed
 
```

Comment:
I guess you forgot to remove
```suggestion
```

Reply:
no, that's there on purpose due to the copy later on

Reply:
oh sorry. looked weird to me in the context of your PR description :D

Reply:
I could change it to:
```
COPY --from=greenbone/community-feed-vts /opt/greenbone/feed/plugins /var/lib/openvas/plugins
```
to get rid of the feed declaration but it looks clunky to me.